src/Sdk/DTWebApi/WebApi/JobEvent.cs

```diff
@@ -189,6 +202,13 @@ public List<ActionsStepTelemetry> ActionsStepsTelemetry
             get;
             set;
         }
+
+        [DataMember(EmitDefaultValue = false)]
```

Comment:
do we have all the `SDK/DTWebApi` changes added to the server side?

Reply:
we do!